### ISSUE TITLE
Login has axe violations

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-org#1933]
- id attribute value must be unique (I believe with the basic preview)
- Heading levels should only increase by one

**Comment by redallen** [comment]
Closing in favor of #2158.
